File: bot.py
import discord
import os

client = discord.Client()

@client.event
async def on_ready():
    print('We have logged in as {0.user}'.format(client))

@client.event
async def on_member_join(member):
    print("New member: " + member.name)
    joinchannel = member.guild.get_channel(880624848559308820)
    await joinchannel.send("Welcome to the server, " + member.mention + "!\nhttps://tenor.com/view/penguins-dance-cool-moves-funny-gif-15070731")

client.run(os.getenv("SLUG_DISCORD_API_KEY"))

File: bot.js
const { Client, Intents } = require('discord.js')
const config = require('./config.js')

const client = new Client({
  intents: [
    Intents.FLAGS.GUILDS,
    Intents.FLAGS.GUILD_MESSAGES,
    Intents.FLAGS.DIRECT_MESSAGES,
    Intents.FLAGS.GUILD_MEMBERS
    // Intents.FLAGS
  ]
});

client.on('ready', () => {
  console.log(`Logged in as ${client.user.tag}!`)
});

//command prefix
const prefix = config.prefix

//funny penguin new member message
client.on('guildMemberAdd', member => {
  console.log('s')
  client.channels.cache.get(config.welcomeChannel).send(`Welcome to SLUG, <@${member.id}>!\nhttps://tenor.com/view/penguins-dance-cool-moves-funny-gif-15070731`)
})

client.login(config.token)